### ISSUE TITLE
Wrong info and link

### ISSUE THREAD

**Issue by chengkai-liu** [XiudingCai/Awesome-Mamba-Collection#5]
Thank you for collecting our paper, "Behavior-Dependent Linear Recurrent Units for Efficient Sequential Recommendation."

However, there are a few corrections to note for this paper:
* The paper has been accepted to CIKM 2024, not WSDM 2024.
* The correct link to the code is https://github.com/chengkai-liu/RecBLR.

Additionally, our paper "Mamba4Rec: Towards Efficient Sequential Recommendation with Selective State Space Models" is accepted to "RelKD@KDD 2024"

I would appreciate it for updating the information. I believe your awesome repo would help us promote our work.

**Comment by XiudingCai** [comment]
Thank you for pointing out these corrections. We sincerely apologize for the oversight. The information has now been updated to reflect the correct conference and code link for "Behavior-Dependent Linear Recurrent Units for Efficient Sequential Recommendation." Additionally, we have included the details for your "Mamba4Rec" paper accepted at RelKD@KDD 2024.

We appreciate your support and are happy to help promote your important work!
